cou/steps/plan.py

```diff
@@ -47,6 +48,8 @@ async def generate_plan(analysis_result: Analysis) -> UpgradeStep:
     )
     for app in analysis_result.apps:
         try:
+            if isinstance(app, AuxiliaryOpenStackApplication):
```

Comment:
Just debug for every charm with its type. Putting condition just for debugging is not a good practice.
`logger.debug("'%s' is an %s charm", app.name, type(app))`

Reply:
I did this to somehow use `AuxiliaryOpenStackApplication`. I've removed this log and added some comments explaining that I need to import this object in order to register the applications.